packages/cli/src/controllers/oauth/abstract-oauth.controller.ts

```diff
@@ -121,17 +121,20 @@ export abstract class AbstractOAuthController {
 		);
 	}
 
-	protected applyDefaultsAndOverwrites<T>(
+	protected async applyDefaultsAndOverwrites<T>(
 		credential: ICredentialsDb,
 		decryptedData: ICredentialDataDecryptedObject,
```

Comment:
we don't need to change it in this PR, but all the decryption code before this call is also leaky abstraction. It'd be nice to (in a subsequent PR) move all calls to `credentialsHelper.getDecrypted` from these files to `credentialsHelper` itself.

Reply:
I agree, i'll create a ticket for that for next bandwidth opening for refactoring